python_modules/dagster/dagster/_core/definitions/declarative_automation/operators/since_operator.py

```diff
@@ -31,21 +31,21 @@ def name(self) -> str:
     def children(self) -> Sequence[AutomationCondition[T_EntityKey]]:
         return [self.trigger_condition, self.reset_condition]
 
-    def evaluate(self, context: AutomationContext[T_EntityKey]) -> AutomationResult[T_EntityKey]:
+    async def evaluate(
+        self, context: AutomationContext[T_EntityKey]
+    ) -> AutomationResult[T_EntityKey]:
         # must evaluate child condition over the entire subset to avoid missing state transitions
         child_candidate_subset = context.asset_graph_view.get_full_subset(key=context.key)
 
         # compute result for trigger condition
-        trigger_context = context.for_child_condition(
+        trigger_result = await context.for_child_condition(
```

Comment:
I'll be annoying and say that this and reset_result can be executed concurrently, so those are also gatherable as well